### PR TITLE
Handle nested data

### PR DESCRIPTION
**Pull request by enzoggqs**
## Motivation

Add view to data with nested data

## Changes

- Subdata view on table mode
- Subdata view on grid mode

## Screenshots

|Before|After|
|---|---|
|image|image|
|image|image|
|image|image|

### CODE REVIEW

**Comment by LeoSilvaGomes** [comment]
Please, ser chevron to the white too

**Review by valdersonjr — APPROVED** [review]
Good job!

File: src/components/Sidebar/RegionDetails/DataSection.tsx [hunk 1]
```diff
@@ -22,26 +22,82 @@ const DataSection: React.FC<MapPropsSectionType> = ({ title, content }) => {
   const hasSelectedDistrict = Boolean(selected);
 
   return (
-    <Collapsible title={title}>
+    <Collapsible className="nesttt" isTitle={true} title={title}>
```
Comment (LeoSilvaGomes) [inline src/components/Sidebar/RegionDetails/DataSection.tsx:25]: What about this `className="nesttt"`?

File: src/components/Sidebar/RegionDetails/DataSection.tsx [hunk 2]
```diff
@@ -22,26 +22,82 @@ const DataSection: React.FC<MapPropsSectionType> = ({ title, content }) => {
...
-            </Styles.ValueContent>
-          ))}
-
-          {!isSelectedOnComparison && hasSelectedDistrict && (
-            <Styles.ValueContent>
-              <p>{selected?.properties.NM_MUN}</p>
-              <MetricDetails district={selected} metric={props} />
-            </Styles.ValueContent>
-          )}
+
```
Comment (LeoSilvaGomes) [inline src/components/Sidebar/RegionDetails/DataSection.tsx:28]: Maybe we can create small components and imports there here!

**Pull request merged**